newdle/client/src/components/Navigator.js

```diff
@@ -21,34 +22,54 @@ export default function Navigator() {
     },
   ];
   const active = useSelector(getStep);
+  const dispatch = useDispatch();
   const activeStep = steps[active - 1];
 
   return (
     <Container>
-      {steps.map((step, index) => (
-        <Step key={index} active={index + 1 === active}>
-          {index + 1}
-        </Step>
-      ))}
+      <ul className={styles['step-box']}>
+        {steps.map((_, index) => (
+          <Step
+            key={index}
+            index={index + 1}
+            active={index + 1 === active}
+            onClick={index + 1 < active ? () => dispatch(setStep(index + 1)) : null}
```

Comment:
I'd allow navigating forward as well, and only disallow going to step 3 if there are no timeslots defined (could add a condition to the corresponding `steps` object that runs the selector, like we do to disable the next button on the step's page)

Reply:
Done.